.github/workflows/conda_and_docs_build.yml

```
@@ -16,12 +16,13 @@ jobs:
     steps:
     - uses: actions/checkout@v1
     - name: conda-build
-      uses: paskino/conda-package-publish-action@v1.4.3
+      uses: paskino/conda-package-publish-action@v2.0.0
```

Test channel priority using this branch: https://github.com/paskino/conda-package-publish-action/compare/master...channel-priority
```suggestion
      uses: paskino/conda-package-publish-action@channel-priority
```

---

From doing this, I learnt that the priority was automatically being set to 'flexible' as expected. It wouldn't let me set it to strict.